src/Pingalot.Core/PingRequestAgent.cs

```diff
@@ -13,20 +16,37 @@ public class PingRequestAgent
 
 		public async Task<PingSession> StartAsync(PingRequestOptions options, CancellationToken cancellationToken)
 		{
+			var ExportFile = options.ExportFile; // https://github.com/TurnerSoftware/Pingalot/issues/6
 			var pingSender = new Ping();
 			var pingOptions = new PingOptions
 			{
 				Ttl = options.TimeTolive
 			};
-			var pingRequests = new List<PingRequest>();
 
-			var buffer = CreateBuffer(options.BufferSize);
+			if (ExportFile != null)
+			{
+				try
+				{
+					SetupExportFile(ExportFile);
+				}
+				catch
+				{
+					// something went wrong with using the provided export file path\filename - so lets setup one local to exe
+					var fileNameDate = DateTime.Now.ToString("yyyy-MM-dd__HH-mm-ss");
+					ExportFile = Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location) + "\\results_" + fileNameDate + ".csv";
+					SetupExportFile(ExportFile);
+				}
```

Comment:
I wouldn't worry about `try`/`catch` here - if we can't setup the export file, just let it fail.

Reply:
@Turnerj  fair enough. 
I am thinking we could accept Export command line parameter without a path and default the path to where the exe runs and the filename with the date time created(same as the catch code above)? Thoughts? I personally liked using a default like that.
Side question - should we do any input validation\sanitation on our command line input? e.g. validate a path is a valid path or an IP is an IP? Maybe something we can add later?